pyomo/contrib/pyros/uncertainty_sets.py

```diff
@@ -348,7 +348,7 @@ def parameter_bounds(self):
         """
         raise NotImplementedError
 
-    def bounding_model(self):
+    def bounding_model(self, config=None):
```

Comment:
You should describe this new argument in the docstring

Reply:
See subsequent commit